python/example/single_cell_detailed.py

```diff
@@ -21,15 +21,11 @@
 
 # (2) Create and populate the label dictionary.
 
-labels = arbor.label_dict()
+# Pre-defined SWC regions
+labels = arbor.label_dict(swc=True)
 
 # Regions:
```

Comment:
This is now a very empty section ;_;

`swc` being what it is, not all users may know what SWC defaults are. Let's add that to the associated tutorial page, with a link to our swc format page (if it isn't already there).

Reply:
I want it _gone_, these things shouldn't be have to remembered let alone typed out again and again.

Reply:
There's now way more text in the docs than we saved by this method ;)